smartsim/experiment.py

```diff
@@ -160,28 +168,45 @@ def __init__(
 
         self.exp_path = exp_path
 
-        self._launcher = launcher.lower()
-
-        if self._launcher == "auto":
-            self._launcher = detect_launcher()
-        if self._launcher == "cobalt":
-            raise SSUnsupportedError("Cobalt launcher is no longer supported.")
-
-        if launcher == "dragon":
-            self._set_dragon_server_path()
-
-        self._control = Controller(launcher=self._launcher)
+        # TODO: Remove this! The contoller is becoming obsolete
+        self._control = Controller(launcher="local")
+        self._dispatcher = settings_dispatcher
+        self._active_launchers: set[LauncherLike[t.Any]] = set()
 
         self.fs_identifiers: t.Set[str] = set()
         self._telemetry_cfg = ExperimentTelemetryConfiguration()
 
-    def _set_dragon_server_path(self) -> None:
-        """Set path for dragon server through environment varialbes"""
-        if not "SMARTSIM_DRAGON_SERVER_PATH" in environ:
-            environ["SMARTSIM_DRAGON_SERVER_PATH_EXP"] = osp.join(
-                self.exp_path, CONFIG.dragon_default_subdir
+    def start_jobs(self, *jobs: Job) -> tuple[LaunchedJobID, ...]:
```

Comment:
just a matter of how you call the function

```py
def list_args(l: list[int]): ...
def tuple_args(t: tuple[int, ...]): ...
def star_args(*a: int): ...

# inline calls
list_args([1, 2, 3])   # Note: 1 arg (list[int]) passed to fn
tuple_args((1, 2, 3))  # Note: 1 arg (tuple[int, int, int]) passed to fn
star_args(1, 2, 3)     # Note: 3 args!! (All `int`s) passed to fn

# From a list variable (equivalent calls to previous section)
list_of_ints = [1, 2, 3]
list_args(list_of_ints)
tuple_args(tuple(list_of_ints))
star_args(*list_of_ints)
```

Reply:
Ohhh I see so this is basically you saying, the user will pass in a Sequence of Jobs, doesnt matter what, might be a list or tuple

Reply:
or hmmm this is actually so a user may do `start_jobs(Job_1, Job_2, Job_3)` instead of` start_jobs([Job_1, Job_2, Job_3])` or `start_jobs((Job_1, Job_2, Job_3))`?

Reply:
> or hmmm this is actually so a user may do start_jobs(Job_1, Job_2, Job_3) instead of start_jobs([Job_1, Job_2, Job_3]) or start_jobs((Job_1, Job_2, Job_3))?

Exactly right!!